cmd/bootstrap/main.go

```diff
@@ -115,7 +116,7 @@ func main() {
 	}
```

Comment:
At present we have auth client 
```go
	auth, authHandler, err := authClient.Setup(envPrefix, cfg.JaegerURL)
	if err != nil {
		logger.Fatal(err.Error())
	}
	defer authHandler.Close()
```
Instead of above code we can have below code , 

```go
	users, usersHandler, err := usersClient.Setup(envPrefix, cfg.JaegerURL)
	if err != nil {
		logger.Fatal(err.Error())
	}
	defer usersHandler.Close()
```
`internal/clients/auth` have default env var `MF_AUTH_GRPC_` 
It should we changed to `MF_USERS_GRPC_` , Since `MF_USERS_GRPC_PORT` is used in users service for gRPC port 
and 
also we can rename the package from `internal/clients/auth` to `internal/clients/users`.